homeassistant/components/template/lock.py

```diff
@@ -115,14 +138,14 @@ def _update_state(self, result):
             return
 
         if isinstance(result, bool):
-            self._state = result
+            self._state = STATE_LOCKED if result else STATE_UNLOCKED
             return
 
         if isinstance(result, str):
-            self._state = result.lower() in ("true", STATE_ON, STATE_LOCKED)
+            self._state = result.lower()
             return
 
-        self._state = False
+        self._state = STATE_UNKNOWN
```

Comment:
Why use unknown state constant? It won't translate to that state. I'd just use `None`.

Reply:
Good call. Will adjust tomorrow after sleep.

Reply:
https://github.com/home-assistant/core/pull/53286